src/testing/consensus.rs

```diff
@@ -207,7 +207,12 @@ impl<D: DataSourceLifeCycle + UpdateStatusData> MockNetwork<D> {
                                 .unwrap(),
                             ConsensusMetricsValue::new(&*data_source.populate_metrics()),
                             hs_storage,
-                            TestAuctionResultsProvider::default(),
+                            MarketplaceConfig {
```

Comment:
this is the only change which we don't have in https://github.com/EspressoSystems/hotshot-query-service/pull/675/files